make.go

```diff
@@ -279,6 +279,26 @@ func runGitCommandIn(dir string, arg ...string) error {
 	return cmd.Run()
 }
 
+func execGitConfig(args ...string) (string, error) {
+	gitArgs := append([]string{"config", "--get", "--null"}, args...)
+	var stdout bytes.Buffer
+	cmd := exec.Command("git", gitArgs...)
+	cmd.Stdout = &stdout
```

Comment:
Instead of dealing with the buffer manually, use `cmd.Output()` instead of `cmd.Run()` below.